app/_landing_pages/service-catalog.yaml

```diff
@@ -45,7 +45,7 @@ rows:
                 Connect external tools like GitHub, PagerDuty, and CI/CD pipelines to map services, dependencies, and metadata.
               icon: /assets/icons/plug.svg
               cta:
-                text: View integrations
+                text: Learn more
```

Comment:
Why "learn more"? That's generally a bad practice in ADA guidelines, as the link isn't descriptive. I know we're not necessarily going out of our way to build in ADA compliance, but changing the text to this hurts more than helps.